docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: '3'
+networks:
+  db-data-net:
+    driver: bridge

```

Comment:
We do usually put networks at the end of file.

Reply:
OK, moved. But it was done like on video and Theory so fix them too...